### ISSUE TITLE
Quick list of possible forced colour mode improvements

### ISSUE THREAD

**Issue by querkmachine** [querkmachine/beeps.website#351]
Boundaries of logo's clickable area aren't visible. 

Could add a border/outline, or alternatively manually specify system colours so that background colour remains present. 

<img width="292" height="166" alt="Image" src="https://github.com/user-attachments/assets/fa728c87-a5b9-4645-ac67-c5b584c5f14f" />

About page icons don't alter to match forced colours. 

<img width="1044" height="322" alt="Image" src="https://github.com/user-attachments/assets/9a29e30f-7723-480e-b179-8a52c87be380" />

Stash items lack bookmark graphic. 

<img width="245" height="181" alt="Image" src="https://github.com/user-attachments/assets/85458011-a8ef-4107-9eba-f7849f166455" />

Code blocks lose syntax highlighting (is this okay though?)

<img width="506" height="534" alt="Image" src="https://github.com/user-attachments/assets/5e697b22-abbd-470f-9bc4-36531b04f4f1" />
